### PR TITLE
update jade website and jobfair details

### PR DESCRIPTION
**Pull request by danrschlosser**
- Add content to JADE website (including new table styles)
- Shrink logos on jobfair site
- fix spacing in the footer
- add images to jobfair / jade

screencapture-localhost-5000-jade-1444159912235
screencapture-localhost-5000-jobfair-1444160224634

### CODE REVIEW

**Comment by danrschlosser** [comment]
review: @samw7823 @eunicekokor @anshulkgupta

File: app/templates/user_pass.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+columbia 116bway
```
Comment (eunicekokor) [inline app/templates/user_pass.txt]: what are these/did you mean to commit this?

File: app/templates/jade.html [hunk 2]
```diff
@@ -2,45 +2,110 @@
...
-                <div class="button blue">
+                <div class="button gray">
                     <a href="http://bit.ly/jade2016">Apply For JADE</a>
                 </div>
             </div>
+            <p>Applications close October 19th.</p>
+        </div>
+        <div class="callout-right">
+            <div class="card">
+                <a href="http://bit.ly/startupcareerfair2015">
```
Comment (eunicekokor) [inline app/templates/jade.html]: remove this line

File: app/templates/jade.html [hunk 3]
```diff
@@ -2,45 +2,110 @@
...
+            <thead>
+                <tr>
+                    <th>Time</th>
+                    <th>Event</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <td>8:30 a.m.</td>
+                    <td>Breakfast: </td>
```
Comment (eunicekokor) [inline app/templates/jade.html]: no colon needed!

**Comment by eunicekokor** [comment]
i love the tabling style, so good :100:

**Pull request merged**